src/utils/Condition/meetsCondition.js

```diff
@@ -46,7 +46,11 @@ const meetsCondition = (condition, value) => {
         return true;
       }
       case 'contains': {
-        return value.toString().includes(compare);
+        if (value){
```

Comment:
```suggestion
        if (value) {
```

Alternatively, this whole thing could simply be:
```
        return value?.toString().includes(compare);
```